src/dot/commons/utils.py

```diff
@@ -52,12 +52,41 @@ def find_files_from_path(path: str, ext: List):
         [
             files.extend(glob.glob(path + "**/*." + e, recursive=True)) for e in ext  # type: ignore
         ]
+        random.shuffle(files)
+
+        # filter
+        if filter is not None:
+            files = [file for file in files if filter in file]
+            print("Filtered files: ", len(files))
 
         return files
 
     return [path]
 
 
+def expand_bbox(
+    bbox, image_width, image_height, scale=None
+) -> Tuple[int, int, int, int]:
+    if scale is None:
+        raise ValueError("margin and scale are mutually exclusive parameter")
```

Comment:
update error message? there is no reference to `margin` here.

Reply:
https://github.com/sensity-ai/dot/pull/36/commits/b1a593d5b212485a3328cf033a043ea807ee0457